File: Searching/Linear_Search/linear_search.php
<?php

// Function for searching
function linear_search($arr, $x) {

    $n = sizeof($arr);
    for ($i = 0; $i < $n; $i++) {
        if ($arr[$i] == $x)
            return $i;
    }
    return -1;
}

$arr = array(10, 30, 8, 40, 11, 1, 9);
$num = 30;

// Function call
$result = linear_search($arr, $num);
if ($result == -1)
    echo "Element is not present in array";
else
    echo "Element is present at index ", $result;

?>

File: Searching/Linear_Search/LinearSearch.java
class LinearSearch {

    // Function for Searching
    public static int linear_search(int arr[], int num) {

        int n = arr.length;
        for (int i = 0; i < n; i++) {

            if (arr[i] == num)
                return i;
        }
        return-1;
        
    }

    public static void main(String[] args) {
        int arr[] = {10, 30, 8, 40, 11, 1, 9};
        int number = 11;

        // Function call
        int result = linear_search(arr, number);
        if (result == -1)
            System.out.println("Element is not present in array.");
        else
            System.out.println("Element is present at index "+result);
    }
}
